aarc_g002_entitlement/__init__.py

```diff
@@ -160,47 +170,24 @@ def __str__(self):
         if not self.is_aarc_g002:
             return self._raw
 
-        return ((
-            '<Aarc_g002_entitlement' +
-            ' namespace={namespace_id}:{delegated_namespace}{subnamespaces}' +
-            ' group={group}{subgroups}' +
-            '{role}' +
-            ' auth={group_authority}>'
-        ).format(**{
-                'namespace_id': self.namespace_id,
-                'delegated_namespace': self.delegated_namespace,
-                'group': self.group,
-                'group_authority': self.group_authority,
-                'subnamespaces': ''.join([',{}'.format(ns) for ns in self.subnamespaces]),
-                'subgroups': ''.join([',{}'.format(grp) for grp in self.subgroups]),
-                'role': ' role={}'.format(self.role) if self.role else ''
-        }))
-
-    def __mstr__(self):
```

Comment:
Notice, that I removed `__mstr__` entirely. Is this used anywhere by anyone? I don't know this dunder method.

Reply:
I'm using that in flaat. I'll move that over at a later point